src/math/lp/lp_primal_core_solver_tableau_def.h

```diff
@@ -107,10 +105,6 @@ unsigned lp_primal_core_solver<T, X>::solve_with_tableau() {
         return 0;
     }
         
-    if ((!numeric_traits<T>::precise()) && this->A_mult_x_is_off()) {
-        this->set_status(lp_status::FLOATING_POINT_ERROR);
-        return 0;
-    }
     do {
```

Comment:
If you delete !numeric_traits<T>::precise() as option, then other occurrences can be removed